src/commands/install.yml

```diff
@@ -7,10 +7,6 @@ parameters:
     description: "Install the Hugo extended binary?"
     type: boolean
     default: true
-  asciidoc:
```

Comment:
I would go ahead and leave this here since you added the functionality. If a user is just using the install command and nothing else in the orb, they would lose this ability now.

Reply:
@dsayling I think it is misleading in my opinion. The reason my CI pipeline fails is because `hugo/build` step is missing `/usr/bin/asciidoc` at run-time. Installing the `asciidoc` package in the `install` step works, but has no actual impact to building or compiling Hugo. I believe it adds an unnecessary step in the `install` step and might mislead someone who thinks it does something that it doesn't.

What do you think?

Reply:
Totally agree - I'm more concerned about the technical API change. A removal of this parameter would technically necessitate a major version bump, however, I'm not sure that's something we should do in this case. Hopefully @KyleTryon can provide a little direction.

Reply:
Understood! I'll resolve this thread and we can continue in the main thread.